docs/man/qdstat.8.adoc

```diff
@@ -412,6 +412,285 @@ The operational status of this auto link:
 lastErr::
 The description of the last attach failure that occurred on this auto link.
 
+qdstat --policy
+~~~~~~~~~~~~~~~
+
+Maximum Concurrent Connections::
+The maximum number of concurrent client connections allowed for this router.
+
+Maximum Message Size::
+The maximum size in bytes of AMQP message transfers allowed for this router as messages enter the
+router network. This limit is applied to transfers over user connections and to transfers to interior
+routers from edge routers.
+
+Enable Vhost Policy::
+Enables the router to enforce the connection denials and resource limits defined in the configured
```

Comment:
defined in the configured vhost policies?

do we need the word 'configured' here?

Reply:
No, we don't. :-)